styles.css

```diff
@@ -0,0 +1,25 @@
+body{
+    font-family: 'Kaushan Script', cursive;
+}
+h1{
+    font-size: 60px;
+}
+h2{
+    font-size: 30px;
+}
+p{
+    font-size: 18px;
+}
+h1{
+    font-weight: bold;
```

Comment:
This reiterates the [default browser styles](https://browserdefaultstyles.com/) for a heading element (they are all bold by default) and, therefore, can be removed:
```suggestion
```